### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by m0cak**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by m0cak — COMMENTED** [review]
approve my pr

**Comment by m0cak** [comment]
test2

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (Modjak) [inline index.html:49]: index comment

Comment (Modjak) [inline index.html:49]: suggestion

Comment (m0cak) [inline index.html:49]: single comment suggestion

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (m0cak) [inline index.html:50]: single test 2

**Pull request merged**